### PR TITLE
Working like a code reviewer

### PR DESCRIPTION
**Pull request by jlcarrascof**
In this milestone, I:

1. Upload my index.html file.
2. Upload my styles.css file.

from my trials project.

### CODE REVIEW

**Comment by jlcarrascof** [comment]
**# Status: REQUIRED CHANGES** ♻️

Hi @jlcarrascof 👋 👋
I'm @jlcarrascof, Your code reviewer,

Good job so far! 🥇
There are some issues that you still need to work on to go to the next project but you are almost there!

image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <p>The man who saved a billion lives</p>
+      <div id="img-div">
+        <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+        <p> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+      </div>
+      <div id="tribute-info">
+        <h4>Here's a time line of Dr. Borlaug's life:</h4>
+      </div>
+      <a href="www.google.com" id="tribute-link" target="_blank">More about Dr. Norman Borlaug</a> 
+    </div>
```
Comment (jlcarrascof) [inline index.html:23]: Maybe you can use some semantic tags in your project.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+img {
+  display: block;
+  max-width: 100%;
+}
```
Comment (jlcarrascof) [inline css/style.css:4]: Use some responsive classes to implement Flexbox, please.

**Review by joel-kalema — COMMENTED** [review]
Hi @jlcarrascof 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that doesn't have a review comment. Kindly check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) about how to give a code review.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jlcarrascof** [comment]
Oh, yes ... Joel @joel-kalema .... My reviews appears as "pending". I'm fixing this issue, right now. Thanks for the info in your link.

Javier.-

**Review by jlcarrascof — COMMENTED** [review]
Good luck implementing these changes.

Javier.-

File: index.html (hunk 1 above)
Comment (joskalenda) [reply index.html:23]: - [ ] Well but not really convinced, you are telling the student to use more semantic tags, why and what is a semantic tag? please be as clear and explanatory as you can so the student will understand what and how important the change is for him.
- [ ] Please be word friendly, use suggestions and polite syntax to make the student feel respected too

File: css/style.css (hunk 2 above)
Comment (joskalenda) [reply css/style.css:4]: - [ ] `Use some ...`, hum this sounds like an order, isn't it? please be friendly and do not be imperative in your sentences, make sure when giving the review comment, you explain what it is important to implement the changes

**Comment by joskalenda** [comment]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @jlcarrascof , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Good inline github feature

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jlcarrascof) [reply index.html:23]: Great job, so far

Remember that is a good practice to use semantic tags, I mean HTML tags that provide meaning and context to the content they surround. Examples, you can include 'header,' 'nav,' and 'footer' replacing some divs in your code. These tags helps your code look more profesional.

File: css/style.css (hunk 2 above)
Comment (jlcarrascof) [inline css/style.css:4]: I have another suggestion for you,

It's a good practice to begin the styles with some properties for the **body section**, for example:

body {

}

and you can consider using some properties like **background-color**, **width**, and **height** that can improve your design.

**Review by jlcarrascof — COMMENTED** [review]
Good luck and if you have any other doubt, I'll be ready to help you. Only use @jlcarrascof to tag me.

Thanks.

**Review by Olamarx — APPROVED** [review]
Hi @jlcarrascof,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Polite comments ✔️
- Reasons for comments ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jlcarrascof** [comment]
Thanks a lot, guys @joskalenda , @joel-kalema and @Olamarx for help me to understand how can I do this job as code reviewer.

Best wishes,

Javier.-

**Pull request merged**